src/Umbraco.Web/PublishedCache/NuCache/DataSource/LazyCompressedString.cs

```diff
@@ -31,6 +33,10 @@ public byte[] GetBytes()
                 throw new InvalidOperationException("The bytes have already been expanded");
             return _bytes;
         }
+        public bool IsDecompressed()
```

Comment:
Maybe nicer as a property? Be nice with some comments explaining what this is for

Reply:
done